src/components/CityWeatherShortcut/CityWeatherShortcut.test.tsx

```diff
@@ -33,7 +34,7 @@ describe('CityWeatherShortcut', () => {
     expect(cityName).toBeInTheDocument();
   });
   it('should checks if url pathname changed from "/" to "/city/Katowice"', async () => {
-    render(<CityWeatherShortcut cityName="Katowice" />);
+    render(<CityWeatherShortcut cityName="Katowice" button={<FavoritesButton cityName="Wroclaw" size={20} />} />);
```

Comment:
**suggestion** I think it should be `button={<FavoritesButton cityName="Katowice" size={20} />} `

Reply:
Right, I just copied/pasted Wroclaw in all tests. My mistake